File: IqbalAhnafFadholi_33852.py
paket_wifi = ["10mb" , "20mb" , "30mb"] # variabel dengan paket wifi yang ada

print("\nList paket wifi yang tersedia : ") # Menampilkan menu paket yang tersedia
for pkt in paket_wifi:
    print(f"- {pkt}")

pilih_paket = input("\nPilih paket anda : ") # variabel dengan input untuk mengisi pilihan paket yang ingin di pilih

def main(): # membuat sebuah fungsi
    for paket in paket_wifi: # perulangan agar paket_wifi terdapat dalam daftar
        if paket == pilih_paket: # untuk mengecek kondisi apakah paket tersedia
            print(f"\nAnda memilih paket {pilih_paket}") # text yang akan muncul jika paket tersedia di daftar
            break # berhenti jika sudah benar kondisinya
    else: # jika paket tidak dalam daftar , maka muncul text tidak tersedia
        print(f"\nPaket tidak tersedia") # text yang akan muncul jika paket tidak tersedia

    
main() # memanggil sebuah fungsi agar berjalan

File: IqbalAhnafFadholi_33852.php
<?php
$paket_wifi = array("10mb" , "20mb" , "30mb"); # variabel paket yang ada
$pilihan = 2; # pilihan paket yang akan di pilih

echo "Pilihan paket yang ada : "; #Menampilkan daftar paket yang ada
foreach ($paket_wifi as $paket) { #Menampilkan daftar paket yang ada
    echo "<br>"."- $paket"; #Menampilkan daftar paket yang ada
}

function cekPaket($pilihan) { # Membuat sebuah fungsi
    if($pilihan === 1) { # Jika pilihan 1 maka memilih paket 10mb
        echo "<br>"."Anda memilih paket 10mb"; # text yang akan muncul
    } elseif ($pilihan === 2) { # jika pilihan 2 maka memilih paket 20mb
        echo "<br>"."Anda memilih paket 20mb"; # text yang akan muncul
    } elseif ($pilihan === 3) { # jika pilihan 3 maka memilih paket 30mb
        echo "<br>"."Anda memilih paket 30mb"; # text yang akan muncul
    } else { # jika yang dipilih tidak ada maka akan error
        echo "<br>"."Error... Paket tidak tersedia"; # text yang akan muncul
    }
}

cekPaket($pilihan) # memanggil fungsi agar kode berjalan

?>